src/screens/FeedScreen/index.js

```diff
@@ -231,6 +231,8 @@ const FeedScreen = (props) => {
       searchHeight={searchHeight}
       bottomArea={bottom}
       isScroll={isScroll}
+      hideThreeDot={false}
+      isSelf={item.is_self}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `hideThreeDot` and `isSelf` props seems fine. However, it's important to ensure that the component receiving these props is properly handling them. If not already done, you should add appropriate PropTypes validation or TypeScript interfaces for these new props in the component definition.

```diff
// In the component definition file
import PropTypes from 'prop-types';

ComponentName.propTypes = {
+  hideThreeDot: PropTypes.bool,
+  isSelf: PropTypes.bool,
  // other prop validations...
};
```
Or if you're using TypeScript:

```diff
// In the component definition file

interface ComponentProps {
+  hideThreeDot?: boolean;
+  isSelf?: boolean;
  // other props...
}
```
Remember to also handle default values for these props if necessary.


<!-- This is an auto-generated comment by OSS CodeRabbit -->